Firebase/InAppMessaging/Runtime/FIRIAMRuntimeManager.m

```diff
@@ -347,6 +337,17 @@ - (void)internalStartRuntimeWithSDKSettings:(FIRIAMSDKSettings *)settings {
                                              activityLogger:self.activityLogger
                                        analyticsEventLogger:analyticsEventLogger];
 
+  self.fetchOnAppForegroundFlow =
```

Comment:
Are there any implications about this being moved to after the `displayExecutor` instantiation?

Reply:
None that I can see. I also tested the end to end flow and there are no changes.